test/test_distributed_models.jl

```diff
@@ -362,14 +362,14 @@ function test_triply_periodic_halo_communication_with_141_ranks(halo, child_arch
         fill!(field, arch.local_rank)
         fill_halo_regions!(field)
 
-        @test all(north_halo(field, include_corners=false) .== arch.connectivity.north)
-        @test all(south_halo(field, include_corners=false) .== arch.connectivity.south)
+        @test @allowscalar all(north_halo(field, include_corners=false) .== arch.connectivity.north)
+        @test @allowscalar all(south_halo(field, include_corners=false) .== arch.connectivity.south)
 
-        @test all(interior(field) .== arch.local_rank)
-        @test all(east_halo(field, include_corners=false) .== arch.local_rank)
-        @test all(west_halo(field, include_corners=false) .== arch.local_rank)
-        @test all(top_halo(field, include_corners=false) .== arch.local_rank)
-        @test all(bottom_halo(field, include_corners=false) .== arch.local_rank)
+        @test @allowscalar all(interior(field) .== arch.local_rank)
+        @test @allowscalar all(east_halo(field, include_corners=false) .== arch.local_rank)
+        @test @allowscalar all(west_halo(field, include_corners=false) .== arch.local_rank)
+        @test @allowscalar all(top_halo(field, include_corners=false) .== arch.local_rank)
+        @test @allowscalar all(bottom_halo(field, include_corners=false) .== arch.local_rank)
```

Comment:
@navidcy are these changes needed for this PR? they don't seem to involve field time series